src/Libraries/RevitNodes/Elements/InternalUtilities/ElementUtils.cs

```diff
@@ -48,5 +51,65 @@ public static void UpdateLevelName(ref string name)
                 name += "(1)";
             }
         }
+
+
+        /// <summary>
+        /// Get a revit parameters value
+        /// </summary>
+        /// <param name="param">Revit parameter</param>
+        /// <returns></returns>
+        public static object GetParameterValue(Autodesk.Revit.DB.Parameter param)
```

Comment:
sure, will do. I assumed the class attribute ([IsVisibleInDynamoLibrary(false)]) would be inherited for all methods and therefore be invisible as well.